src/clusterfuzz/_internal/tests/core/bot/tasks/utasks/progression_task_test.py

```diff
@@ -295,8 +295,7 @@ def test_crash_on_latest(self):
     result = progression_task._check_fixed_for_custom_binary(  # pylint: disable=protected-access
         testcase, testcase_file_path)
     self.assertTrue(result.progression_task_output.crash_on_latest)
-    self.assertEqual(result.progression_task_output.crash_revision,
-                     'app_revision')
```

Comment:
I think this needs to be a string for custom binaries, I wouldn't remove the test for this behavior.